docs/source/manifest_generation.rst

```diff
@@ -0,0 +1,177 @@
+Generate a manifest
+===================
+A **manifest** is a structured file containing metadata that adheres to a specific data model. This page covers different ways to generate a manifest.
+
+Prerequisites
+-------------
+
+**Before Using the Schematic CLI**
+
+- **Install and Configure Schematic**:
+  Ensure you have installed `schematic` and set up its dependencies.
+  See the :ref:`installation` section for more details.
+
+- **Understand Important Concepts**:
+  Understand Important Concepts: Familiarize yourself with key concepts outlined on the :ref:`index` of the documentation.
+
+- **Configuration File**:
+  Learn more about each attribute in the configuration file by referring to the relevant documentation.
+
+
+**Using the Schematic API in Production**
+
+Visit the **Schematic API (Production Environment)**:
+`<https://schematic.api.sagebionetworks.org/v1/ui/#/>`_
+
+This will open the **Swagger UI**, where you can explore all available API endpoints.
+
+
+Generate an empty manifest
+---------------------------------
+
+Option 1: Use the CLI
+~~~~~~~~~~~~~~~~~~~~~
+
+   You can generate a manifest by running the following command:
+
+   .. code-block:: bash
+
+       schematic manifest -c /path/to/config.yml get -dt <your_data_type> -s
+
+   - **-c /path/to/config.yml**: Specifies the configuration file containing your data model location.
+   - **-dt <your_data_type>**: Defines the data type for the manifest (e.g., `"Patient"`, `"Biospecimen"`).
+   - **-s**: Generates a manifest as a Google Sheet.
+
+If you want to generate a manifest as an excel spreadsheet, you could do:
+
+.. code-block:: bash
+
+    schematic manifest -c /path/to/config.yml get -dt <your data type> --output-xlsx <your-output-manifest-path.xlsx>
+
+And if you want to generate a manifest as a csv file, you could do:
+
+.. code-block:: bash
+
+    schematic manifest -c /path/to/config.yml get -dt <your data type> --output-csv <your-output-manifest-path.csv>
+
+Option 2: Use the API
+~~~~~~~~~~~~~~~~~~~~~
+
+1. Locate the `manifest/generate` endpoint.
```

Comment:
Could hyperlink here directly to this endpoint:

https://schematic.api.sagebionetworks.org/v1/ui/#/Manifest%20Operations/schematic_api.api.routes.get_manifest_route

Reply:
Sounds good. I will add that.